Makefile

```diff
@@ -193,7 +193,11 @@ fmt:
 
 .PHONY: vet
 vet:
+	# Default vet
 	$(GO) vet $(GO_PACKAGES)
+	# Custom vet
+	GO111MODULE=on $(GO) build -mod=vendor gitea.com/jolheiser/gitea-vet
```

Comment:
This may need to change depending on whether #10939 gets merged first or not. :slightly_smiling_face:

Reply:
It's not conflicted with #10939 so I think it's not a dependency.

Reply:
We don't need `GO111MODULE=on` any more.